migration/i8.2z/postgresql/202106281200_IDEMPIERE-4863.sql

```diff
@@ -0,0 +1,115 @@
+;
+
+-- Jul 3, 2021, 11:08:01 PM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,DefaultValue,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsAlwaysUpdateable,AD_Column_UU,IsToolbarButton) VALUES (214542,0.0,'Updated','Date this record was updated','The Updated field indicates the date that this record was updated.',200279,'Updated','SYSDATE',29,'N','N','Y','N','N','N',16,0,0,'Y',TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,607,'N','N','D','N','2269b1a5-728e-4405-b636-7d8cf415d9e4','N')
+;
+
+-- Jul 3, 2021, 11:08:01 PM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,IsEncrypted,AD_Reference_ID,AD_Reference_Value_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsAlwaysUpdateable,AD_Column_UU,IsToolbarButton,FKConstraintType) VALUES (214543,0.0,'Updated By','User who updated this records','The Updated By field indicates the user who updated this record.',200279,'UpdatedBy',10,'N','N','Y','N','N','N',30,110,0,0,'Y',TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,608,'N','N','D','N','a1b4cbbb-41d5-4ebb-a9b9-14008fadc7d9','N','D')
+;
+
+-- Jul 3, 2021, 11:08:01 PM CEST
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,PrintName,EntityType,AD_Element_UU) VALUES (203523,0,0,'Y',TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,'AD_Wlistbox_Customization_UU','AD_Wlistbox_Customization_UU','AD_Wlistbox_Customization_UU','D','5f67b77c-08f9-4c20-9463-bedde0025842')
+;
+
+-- Jul 3, 2021, 11:08:02 PM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsAlwaysUpdateable,AD_Column_UU,IsToolbarButton) VALUES (214544,0.0,'AD_Wlistbox_Customization_UU',200279,'AD_Wlistbox_Customization_UU',36,'N','N','N','N','N','N',10,0,0,'Y',TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2021-07-03 23:08:01','YYYY-MM-DD HH24:MI:SS'),100,203523,'N','N','D','N','71dec96d-d4d6-4e4e-b6c9-3091a6784973','N')
+;
+
+	
+SELECT register_migration_script('202106281200_IDEMPIERE-4863.sql') FROM dual;	
```

Comment:
this is not a must but it would be better if you can use current date for migration script.

Reply:
Ok, noted.